src/CommunityToolkit.Maui.UnitTests/xunit.runner.json

```diff
@@ -0,0 +1,4 @@
+{
+    "parallelizeAssembly": false,
+    "parallelizeTestCollections": false
+}
```

Comment:
Just FYI - I added this to ensure the unit tests run serially.

I noticed couple tests were randomly failing because we were setting `Application.Current` (which is a static and used by other tests) to null

Reply:
I was afraid of that, but wasn't sure how to solve it 😦